README.md

```diff
@@ -24,7 +24,8 @@ const combinedComposables = composable0.compose(composable1, composable2, compos
 
 ### Descriptor
 
-**Composable descriptor** (or just **Descriptor**) is a meta data object. It's a collection of properties that contain the information necessary to create an object instance by a *composable*.
+**Composable descriptor** (or just **Descriptor**) is a meta data object. It's a collection of properties that contain
+the information necessary to create an object instance by a *composable*.
```

Comment:
I absolutely agree with you for JavaScript. Not for markdown files.

> If there are 3 lines 100 characters each:
> The people who have soft wraps off will be ok.

Only if their window is > 100 characters wide. Otherwise, the text will seem to jump random intervals unless they figure out there's a horizontal scroll.

> The people who have soft wraps on will be ok.

Again, only if their window is > 100 characters. Otherwise, there will be line breaks in random places and it looks like a mess.

Case in point: GitHub diff displays.

<img width="1716" alt="screen shot 2015-08-07 at 6 38 05 pm" src="https://cloud.githubusercontent.com/assets/364727/9148261/7b8cd8ae-3d33-11e5-8b60-483bdd0e4b7e.png">